ansible_collections/arista/avd/plugins/README.md

```diff
@@ -70,6 +70,36 @@ text : {{ variable }}
 {% endif %}
 ```
 
+## Plugin Tests
+
+Arista AVD provides built-in filters to help extend jinja2 templates
```

Comment:
update this to `Arista AVD provides built-in test plugins to help verify data in Jinja2 templates.`

Reply:
Updated